subsys/zbus/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_sources(zbus.c)
```

Comment:
please create a library (`zephyr_library()`). Also note that it doesn't make sense to add this commit standalone. This means that previous ones introduce a partial feature that can't be compiled.